src/storage/storage_backend.e

```diff
@@ -46,7 +45,7 @@ feature -- Status
 		end
 
 	block_ready: BOOLEAN
-			-- Indicate whether there is a block ready
+			-- Is there a block ready?
```

Comment:
this follows Eiffel style for comments.
A boolean query .. is usually commented with a question.